src/rustdoc-json-types/lib.rs

```diff
@@ -382,6 +382,7 @@ pub enum GenericParamDefKind {
         type_: Type,
         default: Option<String>,
     },
+    Constness,
```

Comment:
If this will stay, please make sure to add some tests for this, and also to bump FORMAT_VERSION at the top of the file